parl/remote/future_mode/utils.py

```diff
@@ -0,0 +1,53 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+import threading
+from collections import namedtuple
+from parl.remote.exceptions import FutureGetRepeatedlyError, FutureFunctionError
+
+
+class FutureObject(object):
+    def __init__(self, output_queue):
+        """Define a new class to avoid user calling the `put` function of output_queue
```

Comment:
Please update the comment for this class. 
We still do not know its purpose in xparl.

Reply:
done